composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
+        "php": "~5.3",
```

Comment:
and i was so happy my whole project did run on php 7 - no more after last vendor update :-1: